en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-inbound-endpoint-example.md

```diff
@@ -0,0 +1,87 @@
+# ISO8583 Inbound Endpoint Example
+
+The ISO8583 inbound endpoint of WSO2 EI acts as a message consumer. The ISO8583 inbound endpoint supported via the WSO2 EI is a listening inbound endpoint that can consume ISO8583 standard messages. 
+
+## What you'll build
+
+This sample demonstrates how the ISO8583 inbound endpoint working as ISO8583 message consumer. In this scenario to generate ISO8583 messages we used sample java client programme.
+
+The ISO8583 inbound endpoint listens on port 5000 and acts as a ISO8583 standard message consumer. When a sample java client connects on port 5000, the ISO8583 inbound endpoint consumes ISO8583 standard messages, converts the messages to XML format, and then injects messages to a sequence into the WSO2 EI.
+
+See [ISO8583 connector configuration](ISO8583-connector-configuration.md) for more information.
+
+The following diagram illustrates all the required functionality of the ISO8583 inbound operations that you are going to build. 
+
+<img src="/assets/img/connectors/ISO8583-inbound-operations.png" title="ISO8583 inbound operations" width="800" alt="ISO8583 inbound operations"/>
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.
+
+{!references/connectors/importing-connector-to-integration-studio.md!}
+
+1. Right click on the created ESB Solution Project and select, -> New -> Inbound Endpoint to create the Inbound Endpoint.
+
+   ```
+   <?xml version="1.0" encoding="UTF-8"?><inboundEndpoint xmlns="http://ws.apache.org/ns/synapse" name="custom_listener" sequence="requestISO" onError="fault" class="org.wso2.carbon.inbound.iso8583.listening.ISO8583MessageConsumer" suspend="false">
+       <parameters>
+           <parameter name="inbound.behavior">listening</parameter>
+           <parameter name="sequential">true</parameter>
+           <parameter name="coordination">true</parameter>
+           <parameter name="port">5000</parameter>
+           <parameter name="headerLength">2</parameter>
+           <parameter name="isProxy">false</parameter>
+       </parameters>
+   </inboundEndpoint>
+   ```
+   Sequence
```

Comment:
Sequence to process the message. 

In this example for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.

Reply:
Updated.